2022/22xxx/CVE-2022-22806.json

```diff
@@ -68,9 +68,14 @@
     "references": {
```

Comment:
('Expecting property name enclosed in double quotes: line 71 column 1 (char 2820)',)